lib/backend/model/ippool.go

```diff
@@ -93,6 +93,7 @@ func (options IPPoolListOptions) KeyFromDefaultPath(path string) Key {
 type IPPool struct {
 	CIDR          net.IPNet `json:"cidr"`
 	IPIPInterface string    `json:"ipip"`
+	IPIPOffSubnet bool      `json:"ipip_off_subnet"`
```

Comment:
oh!  this wasn't supposed to be in here.  I thought I'd removed it.  Oops.

Well i was basically thinking if you didn't want off-subnet IPIP (i.e. you always want IPIP for the pool) then you can just set the subnet to be a /32.